doc/source/scalability/innodb_io.rst

```diff
@@ -31,13 +31,32 @@ System Variables
    :default: True
    :range: True/False
 
-This variable is used to control the ability of the user to set the value of the global |MySQL| variable ``innodb_flush_log_at_trx_commit``.
+This variable enables or disables the effect of the per-session value of 
+the `innodb_flush_log_at_trx_commit` variable. 
 
-If ``innodb_use_global_flush_log_at_trx_commit=0`` (False), the client can set the global |MySQL| variable, using: ::
+The following are the available values:
 
-  SET innodb_use_global_flush_log_at_trx_commit=N
+    * ``1`` - Evaluated as ``TRUE`` and is the default value
+    * ``0`` - Evaluated as ``FALSE``
 
-If ``innodb_use_global_flush_log_at_trx_commit=1`` (True), the user session will use the current value of ``innodb_flush_log_at_trx_commit``, and the user cannot reset the value of the global variable using a ``SET`` command.
+If the global variable  `innodb_use_global_flush_log_at_trx_commit` is
+set to ``1``, the session will always use the current
+global value of `innodb_flush_log_at_trx_commit`. A user cannot reset the value
+with a ``SET`` command for a separate session. Under this mode, changing the
+session variable for ``innodb_flush_log_at_trx_commit`` does not affect
+upstream compatibility.
```

Comment:
maybe "does not affect the value used by the session"? Or simply "this is the upstream compatible mode".

Reply:
Rewritten to "This is the upstream compatible mode."